scripts/popup.js

```diff
@@ -113,14 +113,31 @@ function social_share(eventObj){
     }
 }
 
-function alexa_statistics(eventObj){
-    var open_url="http://www.alexa.com/siteinfo/" + get_clean_url();
-    window.open(open_url, 'newwindow', 'width=1000, height=1000,left=0');
-}
-
-function whois_statistics(eventObj){
-    var open_url="https://www.whois.com/whois/" + get_clean_url();
-    window.open(open_url, 'newwindow', 'width=1000, height=1000,left=0');
+function statistics(eventObj) {  //Common Function for alexa and whois statistics
+    var target=eventObj.target;
+    var id=target.getAttribute('id');
+    var url=get_clean_url();
+    var length=url.length;
+    var start_index="";
+    if(url.includes('https://')){
```

Comment:
You don't need to do all this complex work to remove the `http://` and `https://` prefixes.
You could just use the following regular expression:
`url = url.replace(/^https?:\/\//,'')`
and you are done. Please fix this code.

I suggest you check JavaScript Regular extensions. It will save you a lot of code and time in general.